interface/forms/procedure_order/new.php

```diff
@@ -396,130 +396,179 @@ function validate(f) {
 });
```

Comment:
This page was already updated with bootstrap and modernized. Suggest keeping it as-is